Makefile

```diff
@@ -13,43 +28,17 @@ check-hail:
 	$(MAKE) -C hail/python check
 
 .PHONY: check-services
-check-services: check-auth check-batch check-ci check-gear check-memory \
-  check-notebook check-monitoring check-web-common check-website
-
-.PHONY: check-auth
-check-auth:
-	$(MAKE) -C auth check
-
-.PHONY: check-batch
-check-batch:
-	$(MAKE) -C batch check
-
-.PHONY: check-ci
-check-ci:
-	$(MAKE) -C ci check
-
-.PHONY: check-gear
-check-gear:
-	$(MAKE) -C gear check
-
-.PHONY: check-memory
-check-memory:
-	$(MAKE) -C memory check
-
-.PHONY: check-notebook
-check-notebook:
-	$(MAKE) -C notebook check
-
-.PHONY: check-monitoring
-	$(MAKE) -C monitoring check
-
-.PHONY: check-web-common
-check-web-common:
-	$(MAKE) -C web_common check
-
-.PHONY: check-website
-check-website:
-	$(MAKE) -C website check
+check-services: $(CHECK_SERVICES_MODULES)
+
+.PHONY: check-%
+$(CHECK_SERVICES_MODULES): check-%:
+	$(PYTHON) -m flake8  --config setup.cfg $*/$*
+	$(PYTHON) -m pylint --rcfile pylintrc $*/$* --score=n
+	$(PYTHON) -m mypy --config-file setup.cfg $*/$*
```

Comment:
Note to self as a followup: We should run these linters not just on `$*/$*` but on `$*` + excluding the `sql` directory

Reply:
we really just want to exclude extant SQL files, right?

Reply:
Do the checkers care if we specify a folder that doesn't exist? Maybe a simple fix is `$*/$* $*/test`?